packages/oui-dual-list/src/dual-list.filter.js

```diff
@@ -0,0 +1,11 @@
+export default function (items, sourceFilterKey, property) {
+    return items.filter(item => {
+        let value;
+        if (!property) {
+            value = item;
+        } else {
+            value = property.split(".").reduce((prev, curr) => prev ? prev[curr] : undefined, item);
```

Comment:
Maybe we could use lodash `_.get` ?

Reply:
This method can be imported it like this:
```
import { get } from "lodash";
```

Reply:
how to replace "property.split(".").reduce((prev, curr) => prev ? prev[curr] : undefined, item);
" with  _.get ?

Reply:
In this case `let value = get(item, property, item);` for replacing all the `if` block

Reply:
This could be used for the filter too :)

Reply:
done. used in filter too.